src/Presentation/Presenters/DevelopmentInternalErrorHtmlPresenter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WMDE\Fundraising\Frontend\Presentation\Presenters;
+
+use WMDE\Fundraising\Frontend\Presentation\TwigTemplate;
+
+class DevelopmentInternalErrorHtmlPresenter implements ExceptionHtmlPresenterInterface {
+
+	private TwigTemplate $template;
+
+	public function setTemplate( TwigTemplate $template ): void {
+		$this->template = $template;
+	}
+
+	public function present( \Exception $exception ): string {
+		return $this->template->render( [ 'message' => $exception->getMessage() ] );
```

Comment:
For even more developer convenience, you could also assign the stack trace ([`$exception->getTrace`](https://www.php.net/manual/de/exception.gettrace.php)) and render it.